haskell-src/Concordium/Types/Parameters.hs

```diff
@@ -1611,8 +1611,9 @@ data FinalizationCommitteeParameters = FinalizationCommitteeParameters
       -- |Maximum number of bakers to include in the finalization committee.
       _fcpMaxFinalizers :: !Word32,
       -- |Determining the staking threshold required for being eligible the finalization committee.
-      -- The required amount is given by @total staked ccd / _fcpFinalizerRelativeStakeThreshold@
-      _fcpFinalizerRelativeStakeThreshold :: !Word32
+      -- The required amount is given by @total staked ccd * _fcpFinalizerRelativeStakeThreshold@
+      -- Accepted values are [0,1].
+      _fcpFinalizerRelativeStakeThreshold :: !PartsPerHundredThousands
```

Comment:
The text says "The required amount is given by @total staked ccd * _fcpFinalizerRelativeStakeThreshold@". It should be "total stake in pools". The stake from passive delegation should not be included.